### ISSUE TITLE
Documentation instructions are out of date.

### ISSUE THREAD

**Issue by joehendrix** [leanprover/std4#333]
The README has the build instructions

```
# if you're generating documentation for the first time
> lake -Kdoc=on update
...
# actually generate the documentation
> lake -Kdoc=on build Std:docs
...
> ls build/doc/index.html
build/doc/index.html
```

The instruction `lake -Kdoc=on build Std:docs` reports the error:

```
error: unknown library facet `docs`
```

We should ensure documentation can be built and update the README.

**Comment by digama0** [comment]
I think long term we should remove this build facet and instead move doc-gen to a `lake install` command which can be pointed at projects which don't themselves depend on doc-gen. We should still maintain up to date doc build instructions on the std README though, whatever that happens to be.

**Comment by austinletson** [comment]
Was this issue resolved by #669? I was able to build the docs with the current instructions

**Comment by digama0** [comment]
fixed by #669
